CHANGELOG.md

```diff
@@ -1,5 +1,91 @@
+- Fixed issues with exclusion list for Brave payments. ([#7940](https://github.com/brave/browser-laptop/issues/7940))
+- Fixed new tab for back/forward history. ([#7892](https://github.com/brave/browser-laptop/issues/7892))
+- Fixed styles on notification & update bars. ([#7853](https://github.com/brave/browser-laptop/issues/7853))
+- Fixed re-launching with a maximized window (When re-launching after maximized, window cannot be maximized again once restored down). ([#7825](https://github.com/brave/browser-laptop/issues/7825))
+- Fixed YouTube.com layout when blocking ads. ([#7818](https://github.com/brave/browser-laptop/issues/7818))
+- Fixed audio indicator has extra bottom padding compared to favicon. ([#7815](https://github.com/brave/browser-laptop/issues/7815))
+- Fixed vertical center display of URL bar (Remove "top: 1px" from .urlbarForm>). ([#7805](https://github.com/brave/browser-laptop/issues/7805))
+- Fixed display of text for new tabs. ([#7724](https://github.com/brave/browser-laptop/issues/7724))
+- Fixed private session tab color to always show purple. ([#7720](https://github.com/brave/browser-laptop/issues/7720))
+- Fixed display of session info on tab. ([#7716](https://github.com/brave/browser-laptop/issues/7716))
+- Fixed contribution statement display. ([#7698](https://github.com/brave/browser-laptop/issues/7698))
+- Fixed custom bookmark name text box not clearing when deleted. ([#7691](https://github.com/brave/browser-laptop/issues/7691))
+- Fixed UI on alert dialogs (Replace float:right with flex). ([#7673](https://github.com/brave/browser-laptop/issues/7673))
+- Fixed accidental muting when tabs are small. Sound controls will now be hidden when you have more than 15 tabs. ([#7665](https://github.com/brave/browser-laptop/issues/7665))
+- Fixed re-launching when maximized to keep window maximized. ([#7664](https://github.com/brave/browser-laptop/issues/7664))
+- Fixed favicon size when tabs are small. ([#7656](https://github.com/brave/browser-laptop/issues/7656))
+- Fixed fresh install of Brave advertises Brave properties. ([#7655](https://github.com/brave/browser-laptop/issues/7655))
+- Fixed remember password prompt accidentally displaying password in notification for certain sites. ([#7649](https://github.com/brave/browser-laptop/issues/7649))
+- Fixed menu on Windows - context menu (right click) does not work on nested items. ([#7624](https://github.com/brave/browser-laptop/issues/7624))
+- Fixed Alt+PrtSc key combination is not recognized by Brave. ([#7566](https://github.com/brave/browser-laptop/issues/7566))
+- Fixed Flash discovery for sites if shields are down. ([#7549](https://github.com/brave/browser-laptop/issues/7549))
+- Fixed closed order for tabs in History menu. ([#7548](https://github.com/brave/browser-laptop/issues/7548))
+- Fixed header bar and tab styles (follow up to flat interface). ([#7546](https://github.com/brave/browser-laptop/issues/7546))
```

Comment:
Can we change to `Improved header bar and tab styles (new flat interface look and feel)`